perun/postprocess/regressogram/methods.py

```diff
@@ -0,0 +1,135 @@
+"""
+Module with regressogram computational method and auxiliary methods at executing of this method.
+"""
+
+import numpy as np
+import numpy.lib.function_base as np_bin_selectors
+from click import BadParameter
```

Comment:
Fix: don't use `from module import unit`. It is more error prone and less readable. I would probably even suggest to import numpy as "numpy" at line 5, but in these cases it is sometimes acceptable.